poetry.lock

```diff
@@ -705,7 +705,7 @@ six = ">=1.5"
 [[package]]
 name = "python-kacl"
 version = "0.2.23"
-description = "Pyhton module and CLI tool for validating and modifying Changelogs in \"keep-a-changelog\" format\""
+description = "Python module and CLI tool for validating and modifying Changelogs in \"keep-a-changelog\" format\""
```

Comment:
This is autogenerated file so this will be overridden. I guess the correct place to fix this would be in the `python-kacl` project here: https://github.com/mschmieder/python-kacl/blob/98df1a6c93489e0078deadd47ee80212ffab597e/setup.py#L16